task_2_2/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import "fmt"
+
+// Write max([]string) function that returns the longest word from the slice of strings (the first if there are no more than one)
```

Comment:
Best practice is to start comments with the name of the method, then what it does
https://blog.golang.org/godoc

Reply:
oki, thank you